src/monexamine.cpp

```diff
@@ -117,6 +121,23 @@ bool monexamine::pet_menu( monster &z )
     if( z.has_flag( MF_MILKABLE ) ) {
         amenu.addentry( milk, true, 'm', _( "Milk %s" ), pet_name );
     }
+    if( z.has_flag( MF_SHEARABLE ) ) {
+        bool available = true;
+        if( season_of_year( calendar::turn ) == WINTER ) {
+            amenu.addentry( shear, false, 'S', _( "This animal would freeze if you sheared it now" ) );
+            available = false;
+        } else if( z.has_effect( effect_sheared ) ) {
+            amenu.addentry( shear, false, 'S', _( "This animal is not ready to shear again yet" ) );
```

Comment:
```suggestion
            amenu.addentry( shear, false, 'S', _( "This animal is not ready to shear again yet." ) );
```

Reply:
addentry strings dont require fullstops.

Reply:
Or at least, no other addentry strings elsewhere in the code that ive seen, use them.

Reply:
Though that could be due to conversations like this in the past. no harm in adding them.